lib/src/flutter_emoji_feedback.dart

```diff
@@ -39,19 +40,13 @@ class EmojiFeedback extends StatelessWidget {
     this.enableFeedback = false,
     this.minRating = 1,
     this.maxRating = 5,
-  })  : assert(emojiPreset.length > 0),
+    this.onChangeWaitForAnimation = false,
+    this.tapScale = 0.6,
+  }) /* incompatible with new EmojiPreset class (I think)  : assert(minRating <= maxRating && maxRating <= emojiPreset.emojis.length),
```

Comment:
Same, if something is not relevant anymore, i think it's better to get rid of it